src/renderer/store/handlers/identity.js

```diff
@@ -295,7 +332,13 @@ export const createIdentity = ({ name }) => async (dispatch, getState) => {
         740000
       )
     }
-    await client.rescan()
+
+    setTimeout(async () => {
+      console.log(await client.addresses())
```

Comment:
leftover ?

Reply:
I will keep it for sake of debugging is something goes wrong.